pkg/xray/provider.go

```diff
@@ -102,8 +109,13 @@ func providerConfigure(d *schema.ResourceData, terraformVersion string) (interfa
 	if err != nil {
 		return nil, err
 	}
+
 	_, err = sendUsageRepo(restyBase, terraformVersion)
+	if err != nil {
+		return nil, err
+	}
 
+	err = checkArtifactoryLicense(restyBase)
```

Comment:
Remember to make this optional, per recent RT change - hell, you can probably cherry pick the change

Reply:
@alexhung is it already implemented? I thought it's only a planned change?